pkgs/applications/kde/akonadi-contacts.nix

```diff
@@ -16,10 +16,10 @@ mkDerivation {
   nativeBuildInputs = [ extra-cmake-modules ];
   buildInputs = [
     qtwebengine
-    grantlee grantleetheme
+    grantlee
     kcmutils kdbusaddons ki18n kiconthemes kio kitemmodels ktextwidgets prison
     akonadi-mime kcontacts kmime libkleo
   ];
-  propagatedBuildInputs = [ akonadi ];
+  propagatedBuildInputs = [ akonadi grantleetheme ];
```

Comment:
I can't find out why we this would be required. Also another package is already propagating grantleetheme

Reply:
The CMake files pull it in.